test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -60,6 +60,17 @@ else
             echo "Ignoring $test_file"
         else
             pytest $test_file --ignore=mock/tests/mxnet/test_training_end.py
+            exit_code=$?
+            if [ "$exit_code" -eq "0" ]
+            then
+                echo "SMDebug test succeeded: $test_file"
+            elif [ "$exit_code" -eq "5" ]
+            then
+                echo "SMDebug no tests collected: $test_file"
```

Comment:
When does this happen, and why?

Also, could you add a comment with an explanation in case you have special cases like this?

Finally, in line 79, the script checks `"$?"`, but the exit code variable applies to only the last-executed bash command. This needs to be more concrete.

Reply:
The check below is redundant since we check the result of every test.
I have removed them

Reply:
Could you add a check for the results of the test in line 53?

Reply:
Done